packages/d3fc-series/src/webgl/line.js

```diff
@@ -41,7 +41,7 @@ export default () => {
         const xScale = webglScaleMapper(base.xScale());
         const yScale = webglScaleMapper(base.yScale());
 
```

Comment:
Seting `crossPreviousValueAttribute`, and `mainPreviousValueAttribute` is duplicated, what do you think of something like
```js
const setCrossPreviousValueAttribute = () => crossPreviousValueAttribute.value((d, i) => xScale.scale(base.crossValue()(d, i))).data(data);
const setMainPreviousValueAttribute = () => mainPreviousValueAttribute.value((d, i) => yScale.scale(base.mainValue()(d, i))).data(data);
```
And then calling these instead?

Reply:
Yes but depending on orientation you would need to change the argument of the scaler. So you would need to define 4 const
```
const setCrossPreviousValueAttributeCross = () => crossPreviousValueAttribute.value((d, i) => xScale.scale(base.crossValue()(d, i))).data(data);
const setCrossPreviousValueAttributeMain = () => crossPreviousValueAttribute.value((d, i) => xScale.scale(base.mainValue()(d, i))).data(data);
const setMainPreviousValueAttributeCross = () => mainPreviousValueAttribute.value((d, i) => yScale.scale(base.crossValue()(d, i))).data(data);
const setMainPreviousValueAttributeMain = () => mainPreviousValueAttribute.value((d, i) => yScale.scale(base.mainValue()(d, i))).data(data);
```

If you don't think this is too verbose I can do that